blitzstudios/sleeper-mini
# Adding 6-tab navigation examples.

### Description

This commit adds new code to the example project to show how navigation in 6-tab works. Note that because these navigations are actions, they only work for the instance of the template running from within Sleeper. Tapping on the buttons from within the dev simulator will be a no-op.

### How To Test

1. Update the `selectedSample` in `app.json` to 0. 
2. Connect to the Sleeper App.
3. While running in Sleeper, tap on the navigation buttons. Each one should redirect you to the correct page.
4. Update the `selectedSample` to 1.
5. While connected to sleeper, tap on one of the leagues in the list. A navigation button will open - tap on that to be redirected directly to that league's index page.
